experimental/Origami/src/action.jl

```diff
@@ -0,0 +1,41 @@
+# see https://arxiv.org/pdf/1809.10327.pdf Lemma 5.4
+
+function action_s(o::Origami)
+    h = horizontal_perm(o)
+    v = vertical_perm(o)
+    return origami_disconnected(v^-1, h, degree(o))
+end
+
+function action_t(o::Origami)
+    h = horizontal_perm(o)
+    return origami_disconnected(h, h^-1 * vertical_perm(o), degree(o))
+end
+
+function action_t_inv(o::Origami)
+    h = horizontal_perm(o)
+    v = vertical_perm(o)
+    return origami_disconnected(h, h * v, degree(o))
+end
+
+function action_s_inv(o::Origami)
+    h = horizontal_perm(o)
+    v = vertical_perm(o)
+    return origami_disconnected(v, h^-1, degree(o))
+end
+
+function action_sl2(A::ZZMatrix, o::Origami)
```

Comment:
Since actions in GAP and Oscar are from the right, it would be better to define this as
```suggestion
function action_sl2(o::Origami, A::ZZMatrix)
```

Reply:
(Then again, the current order matches the argument order of `GAP.Globals.ActionOfSL2` so...)